x/lscosmos/keeper/msg_server.go

```diff
@@ -541,3 +541,35 @@ func (m msgServer) JumpStart(goCtx context.Context, msg *types.MsgJumpStart) (*t
 
 	return &types.MsgJumpStartResponse{}, nil
 }
+
+// RecreateICA defines a method for recreating closed ica channels
+func (m msgServer) RecreateICA(goCtx context.Context, msg *types.MsgRecreateICA) (*types.MsgRecreateICAResponse, error) {
+	ctx := sdktypes.UnwrapSDKContext(goCtx)
```

Comment:
Can anyone do this transaction or only the fee address can do it?

Reply:
anyone